engine/src/test/java/io/camunda/zeebe/engine/processing/processinstance/ModifyProcessInstanceRejectionTest.java

```diff
@@ -90,4 +90,48 @@ public void shouldRejectCommandWhenAtLeastOneElementIdIsUnknown() {
             "Expected to activate element but no element found in process '%s' for element id(s): 'B', 'C'"
                 .formatted(PROCESS_ID));
   }
+
+  @Test
+  public void shouldRejectCommandWhenAtLeastOneElementIsInsideMultiInstance() {
+    // given
+    ENGINE
+        .deployment()
+        .withXmlResource(
+            Bpmn.createExecutableProcess(PROCESS_ID)
+                .startEvent()
+                .userTask("A")
+                .subProcess(
+                    "subprocess",
+                    s -> s.embeddedSubProcess().startEvent().manualTask("B").manualTask("C").done())
+                .multiInstance(m -> m.zeebeInputCollectionExpression("[1,2,3]"))
+                .manualTask("task")
+                .endEvent()
+                .done())
+        .deploy();
+    final var processInstanceKey = ENGINE.processInstance().ofBpmnProcessId(PROCESS_ID).create();
+    RecordingExporter.processInstanceRecords(ProcessInstanceIntent.ELEMENT_ACTIVATED)
+        .withProcessInstanceKey(processInstanceKey)
+        .withElementId("A")
+        .await();
+
+    // when
+    final var rejection =
+        ENGINE
+            .processInstance()
+            .withInstanceKey(processInstanceKey)
+            .modification()
+            .activateElement("B")
+            .activateElement("C")
+            .expectRejection()
+            .modify();
+
+    // then
+    assertThat(rejection)
+        .describedAs("Expect that elements with ids 'B' and 'C' are inside multi-instance")
+        .hasRejectionType(RejectionType.INVALID_ARGUMENT)
+        .hasRejectionReason(
+            ("Expected to modify instance of process '%s' with activate instructions but the element(s) with id(s) 'B', 'C'"
+                    + " is inside a multi-instance subprocess. The activation of element(s) inside a multi-instance subprocess is not supported.")
+                .formatted(PROCESS_ID));
```

Comment:
🔧 These lines seem very long. Please wrap them differently, so they fit in our [column limit](https://google.github.io/styleguide/javaguide.html#s4.4-column-limit)

Reply:
If only the formatter would do this for me 😞

Reply:
It can [`reflowLongStrings`](https://github.com/diffplug/spotless/blob/main/plugin-maven/README.md#google-java-format), but enabling this has quite some impact. Additionally, this is not configurable in the google-jave-format IntelliJ-plugin, so the settings would be out of sync. Previously, [this was the reason why we did not enable this](https://confluence.camunda.com/display/ZEEBE/2021-11-04+Team+Meeting). 😞